cranelift/codegen/src/isa/aarch64/lower/isle.rs

```diff
@@ -164,7 +164,6 @@ impl Context for IsleContext<'_, '_, MInst, Flags, IsaFlags, 6> {
     fn integral_ty(&mut self, ty: Type) -> Option<Type> {
         match ty {
             I8 | I16 | I32 | I64 | R64 => Some(ty),
```

Comment:
This should list R32 too, right?

Reply:
Probably, though I'm not sure we should address that on this branch. Would you like to add it in a separate PR?